tests/frameworks/test_fastapi.py

```diff
@@ -366,3 +366,53 @@ def test_custom_header_capture(server):
     assert ("this" == asgi_span.data["http"]["header"]["X-Capture-This"])
     assert ("X-Capture-That" in asgi_span.data["http"]["header"])
     assert ("that" == asgi_span.data["http"]["header"]["X-Capture-That"])
+
+
+def test_response_header_capture(server):
+
+     # The background FastAPI server is pre-configured with custom headers to capture
+    
+    with tracer.start_active_span('test'):
+        result = requests.get(testenv["fastapi_server"] + '/response_headers')
+
+    assert result.status_code == 200
```

Comment:
This TS was unfortunately also full with non-unit test assert, but perhaps we could use `self.assertEqual` and further below.